raw_data/preprocessed/africa/liberia_english.txt

```diff
@@ -0,0 +1,374 @@
+
+Protection of Natural Environmental Areas
+(79) [1] The Agency may, in consultation with the relevant line Ministry, declare by published notice, any area of land, river, lake, wetland or coastal zone as a protected natural environment for the purposes of promoting and preserving specific ecological processes, natural environmental systems, natural beauty or places of indigenous wildlife or the preservation of biological diversity in general. [2] The Agency shall, in consultation with the relevant Line Ministry, issue guidelines and prescribe measures for the management and protection of natural environmental areas. Every County Environment Committee with the assistance of the District Environment Committee shall ensure that the guidelines issued and measures prescribed under this section are implemented especially through voluntary self help activities of the District communities.
+
+Protection of Wild Animals and Birds
+(80) [1] All wild animals and birds and in particular, rare, threatened and endangered species and their habitats shall be preserved and protected in accordance with the guidelines and recommendations made by the Agency after consultation with the Line Ministry. [2] The measures prescribed shall include the declaration by legislature of the wildlife conservation areas: [a] Before the declaration carryout an environmental impact study in accordance with Part III of this Law; [b] Investigate and make a report on the social and ecological consequences of the declaration. [3] A declaration under subsection [1] shall state: [a] Whether the wildlife conservation areas are to be wildlife protected area or wildlife management area; [b] The name and detailed boundary description of the protected area under this section. [4] A wildlife protected area shall be: [a] a national park; [b] a wildlife reserve; [c] nature reserve; [d] any other area as the Line Ministry may declare a wildlife protected area. [5] A wildlife management area shall be: [a] a wildlife sanctuary; [b] a community wildlife area; [c] any other area as the Line Ministry may declare a wildlife management area. [6] The Agency shall prescribe conservation measures as are possible to ensure that communities and persons and wildlife co-exist in the wildlife management area and for wildlife to be protected. [7] The Agency shall prescribe such other measures as may be necessary for wildlife management.
+
+Conservation to Energy and Use of Renewable Sources
```

Comment:
`Conservation to Energy` still needs to be replaced with `Conservation of Energy`.